Documents/idea document 2/ideadocument2.md

```diff
@@ -7,8 +7,38 @@
+These metrics are a useful and structured manner to evaluate wether or not the application is working as intended.
+The MoSCoW specifications also describe the importance of each feature.
+Each feature can then be checked if they are implemented and how well they are working.
+The must haves are the most important features and are the bare minimum for the application to present the findings on the neighborhoods.
+The should haves are expanding features on the basic functionality.
+They can be inspected on how well they filter and modify the presented data to the user.
+The could haves describe extra functionalities that can be implemented after the rest of the application is implemented properly.
+
+Besides functional evaluation of the project by the MoSCoW specifications, the project can also be evaluated by testing how well the application performs in different user scenarios.
+In the following table user scenarios for which Amsterdam In Motion could be used in will be described next to the possible relevant stakeholders.
+
+| Stakeholder         | User Scenario                                                                                                                                                                                                |
+|---------------------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| Local Government    | The government wants to know in what area of the city there is a need for more streetlights for runners. The government wants to know what area would be most suitable and in need of a new sports complex.  |
+| Entrepeneurs        | A company wants to know where to build their new football/hockey store.                                                                                                                                      |
+| Individual Athletes | An athlete wants to know on what time of a day of the week the park is least busy. This way the athlete can plan their exercise accordingly.                                                                 |
+
+## Outlook
+There exist other similar projects that also try to divide a city into neighborhoods based on data collected from social media.
+An example of this is [LiveHoods](livehoods.org).
+Their focus however is different than ours.
+They use checkins from the social platform [Foursquare](https://foursquare.com) to identify certain areas of a city.
+This is heavily based around buildings and points of interest (POI's) in the city.
+This can however be interesting to further expand Amsterdam In Motion.
+For example Foursquare could be used to identify important sports complexes around the city.
+This could help get a better understanding of why an area is popular for a certain sports activity.
+In addition to that a mapping of important sports buildings and POI's can be created.
```

Comment:
Please do not start sentence with they or this, it can be avoided. Furthermore, the sentences are short and not coherent